src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/ConnectHelper.cs

```diff
@@ -178,6 +178,7 @@ private static async ValueTask<SslStream> EstablishSslConnectionAsyncCore(Stream
             return sslStream;
         }
 
+#if !BROWSER_DOES_NOT_SUPPORT_QUIC
```

Comment:
Why are the changes in this file necessary?

Reply:
Yes forgot to clean those up.  Thanks for the catch.

Addressed in recent commit